docker/cytoscape/DockerFile

```diff
@@ -0,0 +1,36 @@
+FROM ubuntu:20.04
+
+# PARAMETERS
+ENV CYTOSCAPE_VERSION 3.8.2
+
+# CHANGE USER
+USER root
+
+# INSTALL JAVA
+RUN apt-get update && apt-get -y install default-jdk libxcursor1 xvfb supervisor wget x11vnc
```

Comment:
Typically we would chain these together as a single RUN, as in line 20.  Can we do that here?

Reply:
It may be possible, I used these three run commands from the cytoscape desktop container https://github.com/cytoscape/docker-cytoscape-desktop/blob/master/Dockerfile#L10